packages/shared/src/components/search/SearchResults/SearchResultsSources.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement } from 'react';
+import { Source } from '../../../graphql/sources';
+import { WidgetCard } from '../../widgets/WidgetCard';
+import { UserHighlight, UserType } from '../../widgets/PostUsersHighlights';
+import { ListItemPlaceholder } from '../../widgets/ListItemPlaceholder';
+
+interface SearchResultsSourcesProps {
+  items: Source[];
+  isLoading: boolean;
+  onSourceClick: (source: Source) => void;
+}
+
+export const SearchResultsSources = (
+  props: SearchResultsSourcesProps,
+): ReactElement => {
+  const { items, isLoading, onSourceClick } = props;
```

Comment:
I see you do this a lot, I don't mind, but why not just do inline in arrow function definition `{ items, isLoading, onSourceClick }: SearchResultsSourcesProps`?

Reply:
I do not like so much destructuring in parameters just because how is formatted, but is not a heavy preference! Should i put there for consistency?

Reply:
I was just wondering why you do it, up to you.